### PR TITLE
Feature/axe accesibility

### PR DESCRIPTION
**Pull request by DrLazer**
accessibility fixes to meet axe requirements

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,158 +1,160 @@
...
-    </div>
-
-    <div class="pic-links__pic" style="background-image: url('<%= asset_pack_path 'media/images/piclink2.png' %>')">
-        <div class="pic-links__pic__mask"></div>
-        <div class="pic-links__pic__salaries">
-            <a href="/life-as-a-teacher/teachers-salaries-and-benefits/">
-                <div class="call-to-action-button">Salaries and benefits  <i class="fas fa-chevron-right"></i></div>
+    <section class="pic-links">
+        <div class="pic-links__pic" style="background-image: url('<%= asset_pack_path 'media/images/piclink1.png' %>')">
+            <a href="/life-as-a-teacher/index">
```
Comment (ethax-ross) [inline app/views/pages/home.html.erb:49]: Unrelated to this change, but this URL looks odd -- why isn't it just `/life-as-a-teacher/`?

File: app/views/pages/mailinglist/registration/step3.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<section class="mailing-reg-main">
+<section class="mailing-reg-main" role="main">
```
Comment (ethax-ross) [inline app/views/pages/mailinglist/registration/step3.html.erb:1]: Could the role not be added at the template level?

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (DrLazer) [reply app/views/pages/home.html.erb:49]: not sure, the content driven part was in place before I joined, its worth flagging

File: app/views/pages/mailinglist/registration/step3.html.erb (hunk 2 above)
Comment (DrLazer) [reply app/views/pages/mailinglist/registration/step3.html.erb:1]: not really, need to be able to control when its needed and when it isnt

**Pull request merged**